[FEATURE] Improvements to start page (#330)
TYPO3-Documentation/TYPO3CMS-Reference-TCA: Documentation/Index.rst

especially between core versions 6.2 to 7 LTS, and again between 7 LTS and 8 LTS. Use the version
selector of this documentation to select the documentation variant that fits.

.. _credits:

**Credits**

The original reference to the TCA was written by Kasper Skårhøj. Subsequent
versions have been updated by François Suter. Several core team members and
other contributors maintained this document over time, see the list of
`contributors on GitHub <https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-TCA/graphs/contributors>`__.

A big "Thank You!" goes to every single one of them.

**Contribution**

For general questions about the documentation get in touch with the
`Documentation Team <https://typo3.org/community/teams/documentation>`__.

> If you find a bug in this manual, please be so kind as to make a change, e.g.

Reviewer: ```suggestion
If you find a bug in this manual, please do not be afraid to suggest a fix, e.g.
```

> by using the "Edit me on GitHub" button in the top right corner.

Reviewer: Do we have/need an annotation/emphasis for the button text?

Author: Do you mean a different style. Correct would be `:guilabel:`, at least that is what we usually use.

Reviewer: Then we should use this. :-)

Author: Ok, coming up.

> Alternatively you can just

Reviewer: ```suggestion
Alternatively you can
```
